apollo-ios/Sources/ApolloAPI/GraphQLOperation.swift

```diff
@@ -53,13 +53,26 @@ public struct OperationDefinition {
   }
 }
 
+/// A unique identifier used as a key to map a deferred selection set type to an incremental
+/// response label and path.
+public struct DeferredFragmentIdentifier: Hashable {
+  let label: String
+  let path: [String]
```

Comment:
rename to fieldPath

Reply:
Done in https://github.com/apollographql/apollo-ios-dev/pull/217/commits/6907fd1420facf2e97af380d500e60edd2b299d2.